package.json

```
@@ -2,17 +2,19 @@
   "name": "solc",
   "version": "0.8.11",
   "description": "Solidity compiler",
-  "main": "dist/index.js",
+  "main": "index.js",
   "bin": {
-    "solcjs": "dist/solc.js"
+    "solcjs": "solc.js"
   },
   "scripts": {
     "build": "tsc",
     "postbuild": "node build/postbuild.js",
     "lint": "eslint --ext .js,.ts .",
     "lint:fix": "eslint --fix --ext .js,.ts .",
     "updateBinary": "node build/clean.js && ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
-    "prepublishOnly": "npm run updateBinary npm && npm run build",
+    "prepublishOnly": "node build/prepublish.js",
+    "prepublish": "npm run updateBinary npm && npm run build && cd dist",
```

```suggestion
    "prepublish": "npm run updateBinary && npm run build",
```
The extra `run` seems like something that should not be there.
Also the final `cd dist` has no effect, right? It can't change directory in your shell.